### PR TITLE
React capstone project

### PR DESCRIPTION
**Pull request by elyor-doniyorov**
### Metrics webapp

**In this PR:👇**

- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions and reducers in React.

### CODE REVIEW

File: src/__test__/DetailsPage.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import DetailsPage from '../components/DetailsPage';
+import store from '../redux/configStore';
+import '@testing-library/jest-dom/extend-expect';
```
Comment (arslanbisharat) [inline src/__test__/DetailsPage.test.js:7]: - [ ] Let's add more test cases in your project, currently, you have only two cases.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,69 @@
...
 
-### `yarn eject`
+- Clone the repo with `https://github.com/elyor-doniyorov/favorite-companies.git`
+- Open your terminal or command line
+- Run `git clone` and paste the link
+- Open the folder with your code editor
+- Create a branch to work on
+- Run `npm install`
+- Run `npm run build`
+- Run `npm run start` to launch the server.
```
Comment (arslanbisharat) [inline README.md:41]: - [ ] Let's add how to run test too in your Readme.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mke2111 — APPROVED** [review]
### Hi @elyor-doniyorov, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**